content/licensing.md

```diff
@@ -15,3 +15,5 @@ Post illustrations, and the LocallyOptimistic logos are the property of LocallyO
 
 Authors retain ultimate ownership and copyright over the content (text and figures) they produce. They may distribute and re-license that content as they see fit. However, by publishing content on LocallyOptimistic, authors irrevocably grant LocallyOptimistic the right to continue to host, publish, and share that content in perpetuity.
 
+If you have questions about sharing or re-using content, please contact us at [contact@locallyoptimistic.com](mailto:contact@locallyoptimistic.com).
```

Comment:
Do we need both `admin@` and `contact@` addresses?  I'm assuming emails get forward to the 4 of us?

Reply:
I actually created a bunch of different emails, all of which *currently* forward to the same google group. I figured we could use multiple in case, in the future, we wanted to forward different message types to different people or set up different filtering rules this would be how we'd do it.

![screen shot 2018-06-13 at 5 50 01 pm](https://user-images.githubusercontent.com/5394529/41380215-570cacdc-6f32-11e8-9ffe-b3faf8b1be50.png)